docs/prow/crier.md

```diff
@@ -0,0 +1,177 @@
+        - --config-path=/etc/config/config.yaml
+        - --dry-run=false
+        volumeMounts:
+        - mountPath: /etc/config
+          name: config
+          readOnly: true
+        - name: slack
+          mountPath: /etc/slack
+          readOnly: true
+      volumes:
+      - name: slack
+        secret:
+          secretName: slack-token
+      - name: config
+        configMap:
+          name: config
+```
+
+Additionally, in order for it to work with Prow you must add the following to your `config.yaml`:
+
+> **NOTE:** `slack_reporter_configs` is a map of `org`, `org/repo`, or `*` wildcard to a set of slack reporter configs.
+
+```yaml
+slack_reporter_configs:
+
+  # Wildcard (i.e. catch-all) slack config
+  "*":
+    # default: None
+    job_types_to_report:
+      - presubmit
+      - postsubmit
+    # default: None
+    job_states_to_report:
+      - failure
+      - error
+    # required
+    channel: my-slack-channel
+    # The template shown below is the default
+    report_template: "Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <{{.Status.URL}}|View logs>"
+
+  # "org/repo" slack config
+  istio/proxy:
+    job_types_to_report:
+      - presubmit
+    job_states_to_report:
+      - error
+    channel: istio-proxy-channel
+
+  # "org" slack config
+  istio:
+    job_types_to_report:
+      - periodic
+    job_states_to_report:
+      - failure
+    channel: istio-channel
+```
+
+The Slack `channel` can be overridden at the ProwJob level via the `reporter_config.slack.channel` field:
+```yaml
+postsubmits:
+  some-org/some-repo:
+    - name: example-job
+      decorate: true
+      reporter_config:
+        slack:
+          channel: 'override-channel-name'
+      spec:
+        containers:
+          - image: alpine
+            command:
+              - echo
+```
+
+## Current Slack notification settings
+
+Crier does not send Slack notifications at all for presubmit jobs.
+
+Reporter config:
+```
+job_types_to_report:
+  - postsubmit
+  - periodic
+  - batch
+```
+
+If you don't want to report postsubmit or periodic jobs to report to Slack channel use `skip_report:true`.
+If the job is still in testing phase we can set `optional: true`.
+
+## Migration from plank for github report
+
+First, you need to disable GitHub reporting in Plank, add the `--skip-report=true` flag to the Plank [deployment](https://github.com/kyma-project/test-infra/blob/master/prow/cluster/components/11-plank_deployment.yaml).
+
+Before migrating, be sure plank is setting the [PrevReportStates field](https://github.com/kubernetes/test-infra/blob/de3775a7480fe0a724baacf24a87cbf058cd9fd5/prow/apis/prowjobs/v1/types.go#L566)
+by describing a finished presubmit ProwJob. Plank started to set this field after commit [2118178](https://github.com/kubernetes/test-infra/pull/10975/commits/211817826fc3c4f3315a02e46f3d6aa35573d22f), if not, you want to upgrade your plank to a version includes this commit before moving forward.
```

Comment:
```suggestion
by describing a finished presubmit ProwJob. Plank started to set this field after commit [`2118178`](https://github.com/kubernetes/test-infra/pull/10975/commits/211817826fc3c4f3315a02e46f3d6aa35573d22f). If it is not being set, you must upgrade your Plank to a version that includes this commit before moving forward.
```
Also, is this only for presubmit ProwJobs?

Reply:
No, it refers to the Plank in general.